draft-ietf-jsonpath-base.md

```diff
@@ -1592,6 +1663,54 @@ Change controller:
 Provisional registration? (standards tree only):
 : no
 
+## Function Extensions {#iana-fnex}
+
+This specification defines a new "Function Extensions sub-registry" in
+a new "JSONPath Parameters registry", with the policy "expert review"
+({{Section 4.5 of -ianacons}}).
+
+The expert is instructed to be frugal in the allocation of very short
+function extension names, keeping them in reverse for applications that
+are likely to enjoy wide use and can make good use of their shortness.
+The expert is also instructed to direct the registrant to provide a
+specification ({{Section 4.6 of -ianacons}}), but can make exceptions,
+for instance when a specification is not available at the time of
+registration but is likely forthcoming.
```

Comment:
This may be common policy for IANA, but the risk is that a spec may never appear and we end up with people "squatting" on function names so others can't register them.

Reply:
Perhaps the function name reservation can have an expiry.

Reply:
Finally, I can reply to this observation.
Copying from below:

Expiry is somewhat unusual for IANA registries (early allocations do expire).
What is the behavior you would expect implementations to exhibit?
The function extension to stop working on the expiry day?

Reply:
@gregsdennis I think those questions are for you. (I wasn't suggesting expiry.)

Reply:
I think the DE instructions are open enough so that the DE can be quite insisting on a specification, so I don't see such a potential for disaster here.